source/develop/release-management/features/virt/virt-v2v-integration.html.md

```diff
@@ -139,6 +139,33 @@ Import from external environments using virt-v2v in terms of the general flow:
 
 10. Unlock VM & disks
 
+##### Import OVA from VMware
+
+Currently oVirt support importing OVA only from VMware hypervisor.
```

Comment:
Xen on RHEL 5.x is not creating ova, it will not work since virt-v2v only support ova from vmware.